Выполнен проект 9го спринта. Первая проверка

Malcevfs opened the pull request.

Malcevfs committed:
Выполнен проект 9го спринта. Первая проверка

Malcevfs committed:
Выполнен проект 9го спринта. Первая проверка

avfyodorov commented on src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java:
```diff
@@ -0,0 +1,62 @@
+    private final HashMap<Integer, Film> filmsStorage = new HashMap<>();
+
+    @PostMapping
+    public Film addFilm(@Valid @RequestBody Film film) {
```
Суффиксы (Фильм, Пользователь) в названии методов можно спокойно убрать. И так понятно к какому типу объекта относится метод.
На Ваше усмотрение.

avfyodorov commented on src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java:
```diff
@@ -0,0 +1,62 @@
+        return filmsStorage.values();
+    }
+
+    public int createId() {
```
А нужен ли этот метод вообще? Не проще ли в нужном месте сделать ++ ??

avfyodorov commented on src/main/java/ru/yandex/practicum/filmorate/model/User.java:
```diff
@@ -0,0 +1,35 @@
+    @NotBlank(message = "empty data in Email")
+    @NonNull
+    @Email(message = "invalid Email")
+    protected String email;
```
Сюда хорошо бы добавить аннотацию.
@pattern(regexp = "^\\\S*$") 
нужно, чтобы пробелов вообще не было.

avfyodorov reviewed (commented):
Добрый день!
Практически всё сделано и сделано хорошо, осталось чуть-чуть поправить.

Commit:
9 проект. Вторая проверка.
Исправлены названия методов котроллера. Добавлена проверка на ввод некорректных символов для login пользователя

Malcevfs commented:
Я исправил замечания, но по незнанию нажал на merge без твоего разрешения =/

avfyodorov reviewed (commented):
Добрый день!
 
Ничего страшного, не беда.  Но в следующий раз чуть- чуть внимательней, хорошо? 
Замечаний нет.
Работа принята.
